packages/backend-storage/src/storage_container_entry_generator.ts

```diff
@@ -60,24 +57,24 @@ export class StorageContainerEntryGenerator
       return amplifyStorage;
     }
 
-    // props.access is the access callback defined by the customer
-    // here we inject the roleAccessBuilder into the callback and run it
-    // this produces the access definition that will be used to create the storage policies
-    const accessDefinition = this.props.access(this.roleAccessBuilder);
-
-    this.validateStorageAccessPaths(Object.keys(accessDefinition));
+    // generate the ssm environment context necessary to access the s3 bucket (in this case, just the bucket name)
+    const ssmEnvironmentEntries =
+      ssmEnvironmentEntriesGenerator.generateSsmEnvironmentEntries({
+        [`${this.props.name}_BUCKET_NAME`]:
+          amplifyStorage.resources.bucket.bucketName,
+      });
 
     // we pass the access definition along with other dependencies to the bucketPolicyArbiter
```

Comment:
really small nit: rename `bucketPolicyArbiter` in the comment to match the variable name

Reply:
updated!